$OpenBSD: patch-src_feature_dirclient_dirclient_c,v 1.1 2021/06/17 10:13:03 tb Exp $

src/feature/dirclient/dirclient.c:1877: error: initializer element is not constant

Index: src/feature/dirclient/dirclient.c
--- src/feature/dirclient/dirclient.c.orig
+++ src/feature/dirclient/dirclient.c
@@ -1873,8 +1873,7 @@ dir_client_decompress_response_body(char **bodyp, size
   /* If we're pretty sure that we have a compressed directory, and
    * we didn't manage to uncompress it, then warn and bail. */
   if (!plausible && !new_body) {
-    const int LOG_INTERVAL = 3600;
-    static ratelim_t warning_limit = RATELIM_INIT(LOG_INTERVAL);
+    static ratelim_t warning_limit = RATELIM_INIT(3600);
     log_fn_ratelim(&warning_limit, LOG_WARN, LD_HTTP,
            "Unable to decompress HTTP body (tried %s%s%s, on %s).",
            description1,
